aboutself.css

```diff
@@ -0,0 +1,260 @@
+* {
+  box-sizing: border-box;
+}
+
+.About {
+  position: absolute;
+  width: 375px;
+  height: 1463px;
+  left: 375px;
+  background: #fff;
+}
+
+.Abouth2 {
+  width: 327px;
+  height: 101px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  display: flex;
+  align-items: center;
+  color: #172b4d;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.Hello {
+  width: 327px;
+  height: 144px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
```

Comment:
i suggest you style your about section using flexbox `display: flex`
It's a project requirement to pass this milestone.